### PR TITLE
Removed the outdated Node.js documentation as requested

### PR DESCRIPTION
**Pull request by flowirtz**
# IMPORTANT

Include some sort of reference to the _official_ "Dockerizing Node.js"-guide by Node.js themselves, or else you might run into frustrated users that otherwise dont find it. 
[Here is the link to their repo of the guide](https://github.com/nodejs/nodejs.org/blob/master/locale/en/docs/guides/nodejs-docker-webapp.md)
[And here is their guide on their website.](https://nodejs.org/en/docs/guides/nodejs-docker-webapp/)
### What I did

I removed the outdated Node.js guide, as recommended by @moxiegirl .
The guide used a more than old version of Node.js (0.1, current is >4.X) which resulted in my builds always ending broken and a lot of frustration while initially using Docker. (that changed now!)
### A picture of a cute animal (not mandatory but encouraged)

Well, thats a though one... Is that one ok?
kinda_cute_dog

### CODE REVIEW

**Comment by FWirtz** [comment]
Ha, signed it just for you @GordonTheTurtle :-)

**Comment by moxiegirl** [comment]
@FWirtz Thanks for doing this but we are going to remove this file. If we include something, we have to maintain it and it rightfully belongs on Node.js site. You can resubmit with the file removed if you like.  Would you like to do that?

**Comment by FWirtz** [comment]
@moxiegirl Ok I can quite understand that. I think a link to their page would be more than appreciated though, because the Docker Docs really were (and are) the "landing page" for me and therefore others probably as well.
Would it be ok if I just put something like "We moved this" on the page or would you rather want me to change the links in the navigation?

**Comment by moxiegirl** [comment]
@FWirtz I am planning on removing more than the Node.js file. If you remove this, I'll add back in links to the files we remove at another point. I have to rearrange some things to make that happen...you wouldn't need to worry about that.

**Comment by FWirtz** [comment]
So you want me to just delete the file? Well, that is simple. :+1:

**Comment by FWirtz** [comment]
@moxiegirl Is that the way you ment it? :-)

**Comment by moxiegirl** [comment]
@FWirtz Easiest ever.  I'll do a redirect for this to the node.js site.  I have to do that in another project.

**Comment by FWirtz** [comment]
Great! Thanks a lot!

**Comment by moxiegirl** [comment]
LGTM @thaJeztah

**Comment by thaJeztah** [comment]
Oh, @FWirtz can you squash your commits? There's three commits now in your PR, and we don't need the "history", so if you can squash them, so that there's only a single commit, that'd be great. If you don't know how; you can find some information in our contributors guide; https://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

LGTM after the commits are squashed

**Comment by FWirtz** [comment]
@thaJeztah I just looked at the guide and I think I will be able to do
that. I am in the Dutch/German timezone however, so I rather do it tomorrow
evening, so in about 20 hours from now. Thanks for all you friendly guys
responses!

Sebastiaan van Stijn <notifications@github.com> schrieb am Di., 19. Apr.
2016 23:36:

**Comment by thaJeztah** [comment]
@FWirtz no worries, no need to hurry; ping me when you had time to update :+1:

**Comment by SvenDowideit** [comment]
LGTM post squash.

**Comment by FWirtz** [comment]
@thaJeztah I finally squashed it!

File: docs/examples/index.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ This section contains the following:
 * [Dockerizing MongoDB](mongodb.md)
 * [Dockerizing PostgreSQL](postgresql_service.md)    
 * [Dockerizing a CouchDB service](couchdb_data_volumes.md)         
-* [Dockerizing a Node.js web app](nodejs_web_app.md)
 * [Dockerizing a Redis service](running_redis_service.md)
 * [Dockerizing an apt-cacher-ng service](apt-cacher-ng.md)
 * [Dockerizing applications: A 'Hello world'](../userguide/containers/dockerizing.md)
+
+<!-- * [Dockerizing a Node.js web app](https://nodejs.org/en/docs/guides/nodejs-docker-webapp/)-->
```
Comment (thaJeztah) [inline docs/examples/index.md]: Oh, think these lines can be removed?

Comment (FWirtz) [inline docs/examples/index.md]: They can, its more of a reminder for you guys, so you dont forget to include some sort of reference to that page... :wink: I'll remove 'em

Comment (thaJeztah) [inline docs/examples/index.md]: There's a redirect added in the global "docs.docker.com" repository :-)

Comment (thaJeztah) [inline docs/examples/index.md]: So it's safe to remove them here. Thanks for thinking along though!

Comment (FWirtz) [inline docs/examples/index.md]: Oh even greater! Ill remove em and then squash again. :blush:

**Comment by FWirtz** [comment]
@thaJeztah had some fun with @GordonTheTurtle and Squashing, should now be finallly good to go...

File: docs/examples/index.md (hunk 1 above)
Comment (thaJeztah) [inline docs/examples/index.md]: awesome! sorry, that change is in this PR: https://github.com/docker/docs.docker.com/pull/123, but it's in a private repository, so it probably gives a 404 for you :blush:

**Comment by thaJeztah** [comment]
@FWirtz excellent! @GordonTheTurtle can be a hand full indeed, so thanks for going through that exercise 👍 

LGTM

**Pull request merged**